src/components/Nav/Nav.js

```diff
@@ -9,8 +9,8 @@ const Nav = () => {
         <NavLink to="/collections" className='nav'>COLLECTIONS</NavLink>
         <NavLink to="/journal" className='nav'>JOURNAL</NavLink>
         <NavLink to="/discover" className='nav'>DISCOVER</NavLink>
+        <p className='user-profile'>User</p>
```

Comment:
Can we move this `<p>` tag back out, I've completely changed this in my branch for the login and logout functionality.

Reply:
Yah, the only reason I moved it in was it was impossible to center the tabs on the page - I'll revert and continue eagerly anticipating your changes 😬